Bank/BankFrontend/src/app/components/pay-form-component/pay-form-component.component.ts

```diff
@@ -0,0 +1,53 @@
+import { HttpClient } from '@angular/common/http';
+import { Component, OnInit } from '@angular/core';
+import { NgForm } from '@angular/forms';
+import { Router } from '@angular/router';
+import { PAY_API } from 'src/environments/environment';
+
+@Component({
+  selector: 'app-pay-form-component',
+  templateUrl: './pay-form-component.component.html',
+  styleUrls: ['./pay-form-component.component.css']
+})
+export class PayFormComponentComponent implements OnInit {
+invalidRegister: number = 0;
+
+  constructor(private router: Router, private http: HttpClient) { }
+
+  ngOnInit(): void {
+  }
+
+  pay(form: NgForm) {
```

Comment:
Što  ovu validaciju nisi radio sa Reaktivnim formama ? Imaš tu ugrađeno da postaviš na bude required.